### ISSUE TITLE
External Usability

### ISSUE THREAD

**Issue by rrwright** [l15k4/scalajs-gremlin-client#2]
I am interested in trying to use your code, but when trying to build it, ran into a sequence of dependency problems that seem to end at a dead end. If you have suggestions on how someone else could build/try your code, I would be very appreciative!

The problems I have are when trying to build the scalajs-gremlin-client there is no published artifact for the `reactive-websocket-client` library:

```
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: com.viagraphs#reactive-websocket-client_sjs0.6_2.11;0.0.3-SNAPSHOT: not found
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
```

Presumably, you have this dependency installed locally. Looking through the viagraphs github profile I find this library which looks relevant: https://github.com/viagraphs/reactive-websocket

However the build is currently failing its tests and you include a caveat that:
`This project is still waiting on accepting a pull request in java-websocket, it'll work with my fork of it only`

Have these issues been resolved? —and only the documentation is out of date? Or (even better!): might this library include external dependencies (or otherwise have them published) so that this library is usable?

Your work looks very interesting! I look forward to investigating it further! Thank you!

**Comment by l15k4** [comment]
Hey, all the scalajs code I wrote is almost 2 years old, I just upgraded scalajs version in all projects a year ago, then I stopped because almost everything is using monifu (now monix) that has changed extensively and upgrading everything to current dependency versions would take me whole weekend. 

Also I made a wrong choice using the java-websocket library which is not maintained anymore... So when I have time I will upgrade everything and switch java-websocket to akka-http websocket, but I cannot promise it will be soon because as I said it would take me at least whole day maybe more which I currently don't have.

The tinkerpop stack also moved on in past 2 years quite a lot, so I think you might be better off using something else or get some inspiration only.

The whole https://github.com/viagraphs was used only for a few months then I moved on to something else.
